code/modules/client/preferences_gear.dm

```diff
@@ -975,6 +975,14 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
 	display_name = "Falling Falcons shoulder patch"
 	path = /obj/item/clothing/accessory/patch/falcon
 
+/datum/gear/misc/patch_uscm/devils
+	display_name = "USCM Solar Devils patch "
+	path = /obj/item/clothing/accessory/patch/devils
+
+/datum/gear/misc/patch_uscm/forecon
+	display_name = "USCM Force Reconnaissance patch"
+	path = /obj/item/clothing/accessory/patch/forecon
```

Comment:
forecon is a bit too special to have marines transferring to the almayer, probably wanna get rid of that one

Reply:
Sorry for the late reponse done

Reply:
Nevermind I did an oopsie, resolved agan